bot/src/discord/show_random_word.js

```diff
@@ -65,6 +69,30 @@ async function showRandomWord(
   );
 
   const firstPage = discordContents[0];
+  if (showStrokeOrder || showExamples) {
+    if (firstPage.embed.fields.length > 1) {
+      firstPage.embed.title += ' (click here for more results)';
+    }
+    firstPage.embed.fields = [firstPage.embed.fields[0]]; // discards the rest of the results
+    await channel.createMessage(firstPage);
+  }
+  if (showStrokeOrder) {
+    const kanji = word.match(/([\u4e00-\u9faf])/g);
+    await Promise.all(kanji.map(async (char) => { // shows the strokes of every kanji
+      const stroke = await createKanjiSearchPage(char, undefined, true);
+      await channel.createMessage(stroke);
+    }));
+    if (!showExamples) return null;
+  }
+  if (showExamples) {
+    const examples = await createExampleSearchPages(word);
+    if (examples.length > 1) {
+      [examples[0].embed.title] = examples[0].embed.title.split(' ('); // removes the '(Page 1 of X)' from the title
+      examples[0].embed.title += ' (click here for more examples)';
+    }
+    const example = examples[0]; // discards the other pages
+    return channel.createMessage(example);
+  }
```

Comment:
It's possible there won't be any examples found. In that case, this `examples[0]` will just be a message saying "No examples found".

Reply:
I think that's less confusing than not showing anything.